src/components/live.ts

```diff
@@ -1103,7 +1106,7 @@ export class FrigateCardLiveJSMPEG extends LitElement {
   /**
    * Refresh the JSMPEG player.
    */
-  protected async _refreshPlayer(): Promise<void> {
+  async _refreshPlayer(): Promise<void> {
```

Comment:
Remove the leading `_` since you are making this method public.

Reply:
done.